modules/process/stacktraces_processlist.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package process
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+)
+
+// WriteProcesses writes out processes to a provided writer
+func WriteProcesses(out io.Writer, processes []*Process, processCount int, goroutineCount int64, indent string, flat bool) error {
+	if goroutineCount > 0 {
+		if _, err := fmt.Fprintf(out, "%sTotal Number of Goroutines: %d\n", indent, goroutineCount); err != nil {
+			return err
+		}
+	}
+	if _, err := fmt.Fprintf(out, "%sTotal Number of Processes: %d\n", indent, processCount); err != nil {
+		return err
+	}
+	if len(processes) > 0 {
+		if err := WriteProcess(out, processes[0], "  ", flat); err != nil {
+			return err
+		}
+	}
+	if len(processes) > 1 {
+		for _, process := range processes[1:] {
+			if _, err := fmt.Fprintf(out, "%s  | \n", indent); err != nil {
```

Comment:
Why exactly the `|`? And why the trailing space?

Reply:
Okay, yeah, the `|` can stay. But the space still annoys me
```suggestion
			if _, err := fmt.Fprintf(out, "%s  |\n", indent); err != nil {
```

Reply:
The space is actually helpful if you reprocess the data that comes out in any way.

It should stay.

Reply:
How is that space useful?

Reply:
it's useful to split by space but I've spent way too long discussing this now and I'm just going to acquiesce.